src/dmd/mars.d

```diff
@@ -635,7 +635,7 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         foreach (i; 1 .. modules[0].aimports.dim)
             semantic3OnDependencies(modules[0].aimports[i]);
 
-        const data = ob.peekSlice();
+        const data = (*ob)[];
```

Comment:
If you explicitly call `opSlice` you don’t need the parentheses or the star.

Reply:
That's why ob should probably be made a ref instead of a pointer. But this is an issue for the caller, not OutBuffer.

Reply:
`ref` can only be used on parameters, not on locals or fields.